pkg/config/viper_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	"sync"
 	"testing"
 
-	"github.com/DataDog/viper"
+	"github.com/spf13/viper"
```

Comment:
Don't we want to stick with our fork?

Reply:
we do with the `replace` in the `go.mod` it is ok to still use the origin name. https://github.com/DataDog/datadog-agent/pull/5292/files#diff-37aff102a57d3d7b797f152915a6dc16R50

Else we have an issue with "go module" that says "github.com/spf13/viper" and "github.com/DataDog/viper" pointing to the same repo "github.com/DataDog/viper"

Reply:
Gotcha! 🙇